ircd/resources.yaml

```diff
@@ -65,13 +70,55 @@ spec:
   - name: irc
     port: 6697
     targetPort: 6697
-  - name: websockets
+  - name: http
+    port: 80
+    targetPort: 80
+  - name: https
     port: 443
     targetPort: 443
   selector:
     app: irc
 ---
 apiVersion: v1
+kind: Service
+metadata:
+  name: webirc
+  labels:
+    app: webirc
+spec:
+  ports:
+  - port: 80
+    targetPort: 443
+  selector:
+    app: webirc
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: webirc-ingress
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    cert-manager.io/cluster-issuer: "letsencrypt-prod"
```

Comment:
Don't want this annotation; we already have the `irc.hashbang.sh` cert in the `irc-certs` secret.